lib/samples/find_closest_facility_from_point/find_closest_facility_from_point_sample.dart

```diff
@@ -173,11 +195,14 @@ class _FindClosestFacilityFromPointSampleState
         _routeGraphicsOverlay.graphics.add(routeGraphic);
       }
     }
-    setState(() => _isRouteSolved = true);
+
+    // Set the route solved flag to true.
+    setState(() => _routeSolved = true);
```

Comment:
Enhancement:

What do you think about also adding a flag for when the route task is in progress and displaying the progress indicator then too? Sometimes it solves really fast for me, but others it takes a little longer and then this `routeSolved` flag isn't updated until after the route task is all finished, so the button stays enabled while it waits. With adding in the progress indicator at the start up of the app, it could be fairly simple to also add this. It could be a useful addition for developers if they have larger route tasks than we do in their apps.

Reply:
This sounds good and easy to implement!